packages/datagovmy-ui/package.json

```diff
@@ -55,11 +54,8 @@
       "i18n": [
         "./types/i18n.d.ts"
       ],
-      "icons": [
-        "./src/icons/index.tsx"
-      ],
-      "icons/agency": [
-        "./src/icons/agency.tsx"
+      "icons/*": [
+        "./src/icons/*"
       ],
```

Comment:
Pls restore to original declaration. Opendosm project is using the icons/index.  If you want to use wildcard:

```json
"icons": [
        "./src/icons/index.tsx"
      ],
      "icons/*": [
        "./src/icons/*.tsx"
      ],

```